logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/MessageRepository.kt

```diff
@@ -372,6 +375,20 @@ class MessageDataSource(
         })
     }
 
+    private fun MessageTarget.toOption(ignoredUsers: List<UserId>) = when (this) {
+        is MessageTarget.Client -> if (ignoredUsers.isNotEmpty()) {
+            MessageApi.QualifiedMessageOption.IgnoreSome(ignoredUsers.map { it.toApi() })
```

Comment:
I think I was lost a bit with this, but now it makes sense to me to go with option 3. Since at this point the info needed to be displayed to the user we have it already, so for `MessageTarget.Client` we can just log and go with `IgnoreAll`.